hermes-management/src/main/java/pl/allegro/tech/hermes/management/domain/detection/InactiveTopic.java

```diff
@@ -18,6 +18,19 @@ InactiveTopic notificationSent(Instant timestamp) {
         this.qualifiedTopicName,
         this.lastPublishedMessageTimestampMs,
         newNotificationTimestampsMs,
-        whitelisted);
+        this.whitelisted);
+  }
+
+  InactiveTopic limitNotificationsHistory(int limit) {
+    List<Long> newNotificationTimestampsMs =
+        notificationTimestampsMs.stream()
+            .sorted((a, b) -> Long.compare(b, a))
```

Comment:
nit: do we really need to sort this list? Timestamps should be order first so we can just take last `limit` elements?

Reply:
I don't want to rely on the current implementation that the timestamps will be sorted. If we ever change anything that changes the order of timestamps then we might get an unexpected behavior and this sorting makes sure we are always safe.